### ISSUE TITLE
I have some issues

### ISSUE THREAD

**Issue by CSer-Tang-hao** [Yuxin-CV/DTN#1]
Hi @Yuxin-CV 
DTN is an inspired work and it's a very good idea. But I have some issues that hopefully will be replied. The experiment setting is not clearly explained in the paper. In your code, I find that your model is trained on the train_val joint splits, not only using the train splits as shown in common protocol. Not as described in the paper that "The hyper-parameters are optimized on the validation set". I also find that your model uses the 224x224 as the input, not 84x84. I don't know if you didn't mention this in your paper deliberately? I think your comparison is unfair. When I corrected the above two points and retried the experiment, I find that the accuracy was very poor. Can you give me some good advice?
Looking forward to your reply!

**Comment by Yuxin-CV** [comment]
Hi @CSer-Tang-hao 
Thank you for your interest in our work.

For your first question, we mentioned in the paper: "The hyper-parameters are optimized on the validation set. After that, it will be merged into the training set for the ﬁnal results." Delta-encoder also merge train set & val set for the final result.

For your second question, all the generation based approaches we compared (Delta-encoder, IDeMe-Net, SalNet) and some other works (Qiao et al.) in Table 1 use 224x224 input.

**Comment by CSer-Tang-hao** [comment]
> Hi @CSer-Tang-hao
> Thank you for your interest in our work.
> 
> For your first question, we mentioned in the paper: "The hyper-parameters are optimized on the validation set. After that, it will be merged into the training set for the ﬁnal results." Delta-encoder also merge train set & val set for the final result.
> 
> For your second question, all the generation based approaches we compared (Delta-encoder, IDeMe-Net, SalNet) and some other works (Qiao et al.) in Table 1 use 224x224 input.

Thanks for your reply! I have no doubt about this excellent work. 
Can you provide the related code and splits for the additional 2 datasets(i.e. CUB&CIFAR100)?
